spec/controllers/rating/applications_controller_spec.rb

```diff
@@ -49,8 +50,11 @@
       end
     end
   end
+
   describe 'GET show' do
-    let(:application) { create :application }
+    let(:application) { FactoryGirl.create(:application) }
+    let!(:student0) { FactoryGirl.create(:student, team: application.team) }
```

Comment:
This is just personal preference, but I don't see the point it writing `let` blocks if they aren't actually used in the test itself. The fact that one needs to use the bang version since the lambda is otherwise never being called is also kind of an indicator for me that these `create` calls should either go into a `before` block or the example itself and do not need to be assigned with names.

However, just a style preference 💅